src/app/admin-panel/_actions/create-equipment.ts

```diff
@@ -15,7 +15,7 @@ export async function createEquipment(formData: FormData) {
       Authorization: `Bearer ${token}`,
     },
     body: JSON.stringify({
-      id: formData.get("id"),
+      id: "hhbhb",
```

Comment:
that probably won't work as expected

Reply:
it is workaround for backend mistake in creating endpoint - it expects id but creates id on backend, so let it be here just for now